.github/workflows/sycl-linux-precommit.yml

```diff
@@ -52,6 +52,22 @@ jobs:
       changes: ${{ needs.detect_changes.outputs.filters }}
       e2e_binaries_artifact: sycl_e2e_bin_default
 
+  # If a PR changes CUDA adapter, run the build on Ubuntu 22.04 as well.
+  # Ubuntu 22.04 container has CUDA 12.1 installed while Ubuntu 24.0 image
+  # has CUDA 12.6.1 installed.
```

Comment:
If I get it right, patches with changes in CUDA adaptor will be tested with both CUDA 12.6.1 and CUDA 12.1 (at least build + LIT?).
In addition, I saw CUDA 12.8 is installed on (at least) one of Windows runners.
Do intentionally use different CUDA versions in different workflows (or different machines)?

@npmiller, FYI.

Reply:
yeah im installing cuda 12.8 on all windows runners